source/2020-05-22-the-ember-times-issue-149.md

```diff
@@ -74,14 +75,11 @@ Check out the [full video on EmberMap](https://embermap.com/topics/what-s-new-in
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Setting up Coveralls for your Ember Addons 💪](http://hangaroundtheweb.com/2020/05/setting-up-coveralls-for-your-ember-addons/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Rajasegar Chandran (@rajasegar)](https://github.com/rajasegar/) blogs about Setting up Coveralls for your Ember Addons. He explains how to setup [ember-cli-code-coverage](https://github.com/kategengler/ember-cli-code-coverage) and [Coveralls](https://coveralls.io/) for your repositories. You can also learn how to use these techniques and make it part of your workflow using Github Actions.
```

Comment:
```suggestion
[Rajasegar Chandran (@rajasegar)](https://github.com/rajasegar/) blogs about setting up Coveralls for your Ember Addons. He explains how to setup [ember-cli-code-coverage](https://github.com/kategengler/ember-cli-code-coverage) and [Coveralls](https://coveralls.io/) for your repositories. Coveralls help you deliver code confidently by showing which parts of your code aren’t covered by your test suite. You can also learn how to use these techniques and make it part of your workflow using Github Actions.
```

I took some verbiage from the Coveralls page in case people aren't familiar with it. What do you think?

Thanks for the writeup! 💯

Reply:
Sounds good @amyrlam.